buildSrc/src/main/groovy/io/micronaut/guides/GuideProjectGenerator.groovy

```diff
@@ -65,32 +72,42 @@ class GuideProjectGenerator implements Closeable {
         String packageAndName = "${basePackage}.${appName}"
         ApplicationType type = ApplicationType.DEFAULT
         String name = metadata.slug
+
         List<GuidesOption> guidesOptionList = guidesOptions(metadata)
         for (GuidesOption guidesOption : guidesOptionList) {
-            List<String> guidesFeatures = metadata.features
-            if (guidesOption.language == Language.GROOVY) {
-                guidesFeatures.remove('graalvm')
-            }
+
             BuildTool buildTool = guidesOption.buildTool
             TestFramework testFramework = guidesOption.testFramework
             Language lang = guidesOption.language
             JdkVersion javaVersion = JdkVersion.JDK_8
```

Comment:
do you want us to generate guides with JDK 8 or should we go to 11 ?

Reply:
Umm... As we are defaulting apps in Launch to JDK11 I think we should go with 11 here too